packages/core/src/connectors/utilities/index.ts

```diff
@@ -33,3 +37,29 @@ export function validateConnectorModule(
     throw new ConnectorError(ConnectorErrorCodes.UnexpectedType);
   }
 }
+
+export const readUrl = async (
+  url: string,
+  baseUrl: string,
+  type: 'text' | 'svg'
+): Promise<string> => {
+  if (!url) {
+    return url;
+  }
+
+  if (type !== 'text' && url.startsWith('http')) {
```

Comment:
Better check with URL class and wrap in `try catch` block

Reply:
But we just want to "skip", we are not using the url here.

Reply:
I see... So we don't support reading SVG from an online source?

Reply:
Yeah